spec/support/fetchable_behavior.rb

```diff
@@ -136,3 +173,45 @@
     end
   end
 end
+
+shared_examples_for 'a record with an indexable url' do
+  describe 'validations' do
+    it { is_expected.to allow_value("http://some.site.gov/url").for(:url) }
```

Comment:
This spec should actually be in the `shared_examples_for 'a record with a fetchable url'` section.